prefixdb_test.go

```diff
@@ -20,6 +26,143 @@ func mockDBWithStuff(t *testing.T) DB {
 	return db
 }
 
+func taskKey(i, k int) []byte {
+	return []byte(fmt.Sprintf("task-%d-key-%d", i, k))
+}
+
+func randomValue() []byte {
+	b := make([]byte, 16)
+	_, err := rand.Read(b)
+
+	if err != nil {
+		return nil
+	}
+
+	return b
+}
```

Comment:
We can simplify this as [rand.Read](https://pkg.go.dev/math/rand#Read) never reports an error:

```suggestion
func randomValue() []byte {
	b := make([]byte, 16)
	rand.Read(b)
	return b
}
```